chia/full_node/weight_proof.py

```diff
@@ -593,7 +595,7 @@ def get_fork_point_no_validations(self, weight_proof: WeightProof) -> Tuple[bool
         if len(weight_proof.sub_epochs) == 0:
             return False, uint32(0)
         summaries, sub_epoch_weight_list = _validate_sub_epoch_summaries(self.constants, weight_proof)
-        if summaries is None:
+        if summaries is None or sub_epoch_weight_list is None:
```

Comment:
here and on line 571, these used to be assumptions. Would it make more sense to have asserts instead of checks like this?

Reply:
yes asserting would be better